gm4_holographic_tags/data/holographic_tags/functions/init.mcfunction

```diff
@@ -0,0 +1,49 @@
+#announce module installation
+tellraw @a[gamemode=creative] [{"translate":"%1$s","with":["[GM4]: ",{"translate":"text.gm4.prefix"}]},{"translate":"%1$s","with":["Installing Holographic Tags...",{"translate":"text.gm4.modules.update.installing","with":[{"translate":"module.gm4.holographic_tags"}]}]}]
+execute unless entity @a[limit=1] run say GM4: Installing Holographic Tags...
+
+#declare and initialise scoreboards and settings
+scoreboard players set update_happened gm4_up_check 1
+scoreboard players set holographic_tags gm4_modules 1
+scoreboard players set holographic_tags gm4_clock_tick 0
+
+team add gm4_holo_white
```

Comment:
Why is this `gm4_holo_white` and not `gm4_ht_white`, like the rest?

Reply:
It was a typo; gm4_holo was the original prefix, but some names were too long, like light blue, so I changed it to ht, but I guess I forgot to change the white one. I didn't notice cuz white is the default, so it didn't change anything while I was testing.